### PR TITLE
created api calls that dynamically populate a sample homepage

### PR DESCRIPTION
**Pull request by HibiAdelei**
the html and css can be largely ignored, this is just me pushing my js code for the api calls

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+buildOMDBApiCall();
+
+
+
+var container = document.createElement('div');
+container.setAttribute('class', 'container');
+
+test.appendChild(container);
+
+var request = new XMLHttpRequest();
```
Comment (LemonPocky) [inline script.js]: Any reason to use XMLHttpRequest() here instead of fetch()?

**Review by LemonPocky — COMMENTED** [review]
Make sure you git pull before you push and move script.js into the assets/js folder.

File: script.js (hunk 1 above)
Comment (HibiAdelei) [reply script.js]: not in particular, fetch is fine

File: assets/js/script.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    fetch('https://ghibliapi.herokuapp.com/films')
+    .then(response => {
+        return response.json();
+    })
+    //returns movie data as array
+    .then(data => {
+        console.log(data);
+    });
+}
+apicall();
```
Comment (LemonPocky) [inline assets/js/script.js]: Let's group all of the code outside of a function into an init() function for readability.

File: assets/js/script.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    //   card.appendChild(img);  
+    });
+  } else {
+    var error = document.createElement('marquee');
+    error.textContent = `pain`;
+    test.appendChild(error);
+  }
+}
+request.send();
+}
```
Comment (LemonPocky) [inline assets/js/script.js]: Indentation looks kind of off here. Maybe double check your brackets, so it's easy to tell where request.send() belongs to.

File: assets/js/script.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+init();
```
Comment (LemonPocky) [inline assets/js/script.js]: Put global code such as init(); at the bottom of the file.

**Review by LemonPocky — CHANGES_REQUESTED** [review]

File: assets/js/script.js [hunk 5]
```diff
@@ -1,3 +1,83 @@
...
+        return response.json();
+    })
+    //returns movie data as array
+    .then(data => {
+        console.log(data);
+    });
+}
+
+
+// ======= main
```
Comment (LemonPocky) [inline assets/js/script.js]: Remove this line, it's an artifact from the merge conflict.

File: assets/js/script.js [hunk 6]
```diff
@@ -1,3 +1,83 @@
+// ======= apicalls
```
Comment (LemonPocky) [inline assets/js/script.js]: Remove this line, it's an artifact from the merge conflict.

**Review by LemonPocky — CHANGES_REQUESTED** [review]

**Review by HibiAdelei — COMMENTED** [review]
went ahead and removed those lines as requested

File: assets/js/script.js [hunk 7]
```diff
@@ -1,3 +1,80 @@
...
+        //   img.src = movie.poster;
+        container.appendChild(card);
+        card.appendChild(h1);
+        card.appendChild(p);
+        //   card.appendChild(img);  
+      });
+          
+    } else {
+           var error = document.createElement('marquee');
+           error.textContent = `pain`;
```
Comment (LemonPocky) [inline assets/js/script.js]: lmao

Comment (LemonPocky) [reply assets/js/script.js]: Let's change this to something more descriptive or write a TODO in case we forget to change it later.

**Review by HibiAdelei — COMMENTED** [review]
done

**Review by LemonPocky — CHANGES_REQUESTED** [review]

**Review by HibiAdelei — COMMENTED** [review]
fixed error handler to be a bit more descriptive

**Review by LemonPocky — APPROVED** [review]

**Pull request merged**